app/Models/Course.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Course extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
```

Comment:
Having multiple array entries on one line is not bad, but how you implemented it looks a bit messy. It is better to either have each entry on a separate line or consistently apply some rule that determines how many entries are allowed on one line. Commonly used rule is for example "fill up to max 80 characters line width".